src/main/res/values/strings.xml

```diff
@@ -45,4 +45,9 @@
 	<string name="promptChooseImage">Choose image</string>
 
 	<string name="spnCompressingImage">Compressing image…</string>
+
+	<string name="geolocationPermissionsTitle">Location permissions</string>
+	<string name="geolocationPermissionsDescription">This app would like to record your location with your report submissions.</string>
```

Comment:
What do we do for i18n in the Android app? Obviously we can't (or at least, won't) bake in values for all the unofficial languages that might be found in CHT configs, but how about the "core" languages we support? Also, @garethbowen I know you're working on just what that means, so pending the outcome of that investigation, this may be a place to cover when the time comes. For now I'm curious of your thoughts.

Reply:
Right now the product team doesn't do anything. I'll add it as part of the discussion around supported languages because I agree it's something that needs to be solved. Because each project builds its own flavour the strings can be replaced by whatever they want, but I don't think we support multiple languages in the android container. Historically we haven't had much to translate, but the number is growing slowly so now is a good time to revisit this. Are you happy for this discussion and solution to happen outside of this PR?